tools/analyzer_plugin/lib/src/diagnostic/missing_cascade_parens.dart

```diff
@@ -90,7 +91,7 @@ class MissingCascadeParensDiagnostic extends DiagnosticContributor {
         } else if (isVoidUsage && parent is InvocationExpression) {
           invocation = parent;
         }
-        debug.log('invocation : ${invocation?.toSource()}');
+//        debug.log('invocation : ${invocation?.toSource()}');
```

Comment:
are these logs still useful?

Reply:
Since this diagnostic still needs a bit of work, I left them in just in case they'll be helpful in the future